dtls/Makefile

```diff
@@ -7,13 +7,13 @@ LIBS     = -L$(LIB_PATH)/lib -lm
 # option variables
 DYN_LIB         = -lwolfssl
 STATIC_LIB      = $(LIB_PATH)/lib/libwolfssl.a
-DEBUG_FLAGS     = -g -DDEBUG
+DEBUG_FLAGS     = -g3 -DDEBUG -O0
 DEBUG_INC_PATHS = -MD
 OPTIMIZE        = -Os
 
 # Options
-#CFLAGS+=$(DEBUG_FLAGS)
-CFLAGS+=$(OPTIMIZE)
+CFLAGS+=$(DEBUG_FLAGS)
```

Comment:
Should this be changed by default?

Reply:
No I committed this by mistake. Will revert it.

Reply:
Reverted.